manager-bundle/src/DependencyInjection/ContaoManagerExtension.php

```diff
@@ -15,14 +15,30 @@
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
-use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\HttpKernel\DependencyInjection\ConfigurableExtension;
 
-class ContaoManagerExtension extends Extension
+class ContaoManagerExtension extends ConfigurableExtension
 {
+    /**
+     * @var array
+     */
+    private static $managerPaths = [
+        'contao-manager.phar.php',
```

Comment:
I think the `contao-manager.phar.php` is the only officially supported name, we don't need to test for any other.

Reply:
Nope, that's just the default/documented way. If you don't wanna follow the default, just add the file name to your config 🙃